packages/x-components/static-docs/build-search-ui/README.md

```diff
@@ -43,24 +44,14 @@ numerous components to choose from, and the catalog evolves quickly with new exp
 Check out the **[open source project in GitHub](https://github.com/empathyco/x)**.
 
 Interested in learning more about how Interface&nbsp;X works? Discover more about its
-**[architecture](x-architecture/README.md)**.
-
-## Integrate Interface X Archetype
-
-Once you have finished developing or extending your search interface using the
-Interface&nbsp;X&nbsp;Archetype project you will probably want to integrate it into your current
-store. The integration of an existing Archetype into your project is pretty straightforward. You
-will need to load the generated Javascript into your website and initialise it.
-
-To start integrating an existent Archetype into your project please visit
-[extended guide about Archetype integration](web-archetype-integration-guide.md).
+**[architecture](web-x-architecture.md)**.
 
 ## Develop with Interface X
 
-You can use the Interface X in a project in two ways:
+You can use the Interface&nbsp;X in a project in two ways:
 
 - using the separate **Interface&nbsp;X&nbsp;Archetype** project, an out-of-the-box project with all
-  you need to get up and running fast, or
+  you need to get up and running fast
 - integrating the individual **Interface&nbsp;X&nbsp;Components** library in your Vue project for a
```

Comment:
Should say here `importing` instead of `integrating`. I am suggesting this because we use integrate for the other section, and here we are inside developing...

```suggestion
- importing the individual **Interface&nbsp;X&nbsp;Components** library in your Vue project for a
```